src/CitizenProject/CitizenProjectCommand.php

```diff
@@ -185,4 +202,21 @@ public function setCitizenProject(CitizenProject $citizenProject): void
     {
         $this->citizenProject = $citizenProject;
     }
+
+    public function getCommittees(): Collection
+    {
+        return $this->committees;
+    }
+
+    public function setCommittees(iterable $committees): void
```

Comment:
There is no need for getter/setter if the property is public. I think the property should be made private.

Reply:
✔️